packages/wcg-text-encoding/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@nativescript/wcg-text-encoding",
+  "version": "1.0.0",
+  "description": "Add a plugin description",
```

Comment:
Worth filling in the descriptions and the repository fields, etc. for each of these